README.md

```diff
@@ -2,4 +2,76 @@
+Make [spaCy](https://spacy.io) and [Thinc](https://thinc.ai) **up to 8 &times; faster** on macOS by calling into Apple's native
+libraries.
+
+## 🏫 Motivation
+
+Matrix multiplication is one of the primary operations in machine learning.
+Since matrix multiplication is computationally expensive, using a fast matrix
+multiplication implementation can speed up training and prediction
+significantly.
+
+Most linear algebra libraries provide matrix multiplication in the form of the
+standardized
+[BLAS](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) `gemm`
+functions. The work behind scences is done by a set of matrix multiplication
+kernels that are meticulously tuned for specific architectures. Matrix
+multiplication kernels use architecture-specific
+[SIMD](https://en.wikipedia.org/wiki/SIMD) instructions for data-level parallism
+and can take factors such as cache sizes and intstruction latency into account.
+[Thinc](https://github.com/explosion/thinc) uses the
+[BLIS](https://github.com/flame/blis) linear algebra library, which provides
+optimized matrix multiplication kernels for most x86_64 and some ARM CPUs.
+
+Recent [Apple Silicon](https://en.wikipedia.org/wiki/Apple_silicon) CPUs, such
+as the [M-series](https://en.wikipedia.org/wiki/Apple_silicon#M_series) used in
+Macs, differ from traditional x86_64 and ARM CPUs in that they have a separate
+matrix co-processor(s) called AMX. Since AMX is not well-documented, it is
+unclear how many AMX units Apple M CPUs have. It is certain that the (single)
+performance cluster of the M1 has an AMX unit and there is [empirical
+evidence](https://twitter.com/danieldekok/status/1454383754512945155?s=20) that
+both performance clusters of the M1 Pro/Max have an AMX unit.
+
+
+Even though AMX units use a set of [undocumented
+instructions](https://gist.github.com/dougallj/7a75a3be1ec69ca550e7c36dc75e0d6f),
+the units can be used through Apple's
+[Accelerate](https://developer.apple.com/documentation/accelerate) linear
+algebra library. Since Accelerate implements the BLAS interface, it can be used
+as a replacement of the BLIS library that is used by Thinc. This is where the
+`thinc-apple-ops` package comes in. `thinc-apple-ops` extends the default Thinc
+ops, so that `gemm` matrix multiplication from Accelerate is used in place of
+the BLIS implementation of `gemm`. As a result, matrix multiplication in Thinc
+is performed on the fast AMX unit(s).
+
+## ⏱ Benchmarks
+
+Using `thinc-apple-ops` leads to large speedups in prediction and training on
+Apple Silicon Macs, as shown by the benchmarks below.
+
+### Prediction
+
+This first benchark compares prediction speed of the `de_core_news_lg` spaCy
+model between the M1 with and without `thinc-apple-ops`. Results for an Intel
+Mac Mini and AMD Ryzen 5900X are also provided for comparison. Results are in
+words per second. In this prediction benchmark, using `thinc-apple-ops` improves
+performance by *4.3** times.
```

Comment:
```suggestion
performance by **4.3** times.
```

(Or was this supposed to be italics with `*` for footnote that doesn't exist yet?)

Reply:
It was supposed to be italics, but my editor added one asterisk too many when closing the italics. But maybe bold is even better.